# Algortihm Parameters
alpha: 0.1
gamma: 0.8
epsilon: 0.9
epsilon_discount: 0.999 # 1098 eps to reach 0.1
nepisodes: 100000
nsteps: 1000

# Environment Parameters
desired_pose:
    position:
      x: 0.0
      y: 0.0
      z: 1.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.0

max_vel: 1.0   # in meters
min_vel: -1.0   # in meters
max_acc: 1.0       # in rads
min_acc: -1.0
max_jerk: 1.0
min_jerk: -1.0
max_snap: 1.0
min_snap: -1.0

max_incl: 0.785

running_step: 0.02   # in seconds

done_reward: -1000.0 # reward
alive_reward: 100.0 # reward


weight_r1: 1.0 # Weight for joint positions ( joints in the zero is perfect )
weight_r2: 0.0 # Weight for joint efforts ( no efforts is perfect )
weight_r3: 1.0 # Weight for contact force similar to desired ( weight of monoped )
weight_r4: 1.0 # Weight for orientation ( vertical is perfect )
weight_r5: 1.0 # Weight for distance from desired point ( on the point is perfect )
